eth/MinerAux.h

```diff
@@ -101,8 +101,9 @@ class MinerCLI
 		BasicAuthority::init();
 	}
 
-	bool interpretOption(int& i, int argc, char** argv)
+	bool interpretOption(size_t& i, const vector<string>& argv)
```

Comment:
`const` should be after type name according to our coding style, so should be `vector<string> const&`